tests/config/offensive-ngrams.txt

```diff
@@ -0,0 +1 @@
+offensive angry war violance evil adversity bad break bummer calamity cataclysm catastrophe disaster downer drag evil hard knocks hard times hardship jam misery misfortune mishap storm cloud tragedy tribulation trouble unpleasantnes
```

Comment:
This is not as discussed. The model file should contain (class_id, ngram, count) triples.